core/drivers/crypto/se050/crypto.mk

```diff
@@ -0,0 +1,70 @@
+ifeq ($(CFG_NXP_SE05X), y)
+# Enable the crypto driver
+CFG_CRYPTO_DRIVER = y
+CFG_CRYPTO_DRIVER_DEBUG ?= 0
+
+# SE050 initialization
```

Comment:
Could you explicit here what the several config switches do. Some are not obvious as `CFG_CORE_SE05X_SCP03_EARLY`.

Reply:
can you give me similar examples for this kind of documentation and which ones would you need to get clarified? I'd rather not add more than necessary.
I think this one that you mentioned is clear (could be just in my head): it configures SCP03 Early....early, as with TA, means before REE is available.